docs/using.rst

```diff
@@ -655,8 +612,106 @@ The following files are available:
 
 .. _hooks:
 
-Pre and Post Validation Hooks
-=============================
+Hooks
+-----
+
+The ``renew`` and ``run`` commands include hooks for running system commands or scripts before or after a certificate is
+renewed. For example, if you have a single certificate obtained using
+the standalone_ plugin, you might need to stop the webserver
+before renewing so standalone can bind to the necessary ports, and
+then restart it after the plugin is finished. Example::
+
+  certbot certonly --standalone --pre-hook "service nginx stop" --post-hook "service nginx start"
+
+Hooks specified in the creation of a cert will be saved in that certificate's
+renewal configuration file, and will be called each time renewal for that
+certificate is attempted.
+
+Alternately you can specify hooks as arguments with ``certbot renew`` either on
+the command line or in your crontab. Example::
+
+  certbot renew --pre-hook "service nginx stop" --post-hook "service nginx start"
+  
+.. note:: Any certificates renewed this way will have the hook values in their
+   renewal configuration files overwritten with the arguments specified in this
+   renew command. Certificates for which renewal is not attempted will have the
+   hook values in their renewal configuration files unchanged.
+   
+Similarly global hooks can be specified in the (optional) ``cli.ini``
+configuration file. For more information on configuration files, please
+refer to config-file_.
+
+If a hook exits with a non-zero exit code, the error will be printed
+to ``stderr`` but renewal will be attempted anyway. A failing hook
+doesn't directly cause Certbot to exit with a non-zero exit code, but
+since Certbot exits with a non-zero exit code when renewals fail, a
+failed hook causing renewal failures will indirectly result in a
+non-zero exit code. Hooks will only be run if a certificate is due for
+renewal, so you can run the above command frequently via ``certbot renew`` 
+without unnecessarily stopping your webserver.
+
+Unique ``--pre-hook`` and ``--post-hook`` hooks run before and after every renewal
+attempt. If two certificates have identical pre and post hooks, they will be run
+only once. Meaning renewal for the two certificates would follow this form:
+pre-hook is run, renewal for cert 1 is run, renewal for cert 2 is run, post-hook
```

Comment:
nit: I'd rephrase this to something like:

"Your pre-hook is run, certificate 1 is renewed, certificate 2 is renewed, then your post-hook is run."

to avoid confusion with the now deprecated renew hook.